frontends/web/src/locales/en/app.json

```diff
@@ -24,7 +24,7 @@
     "fiatBalance": "Fiat Balance",
     "name": "Account Name",
     "noAccount": "There are no accounts to show.",
-    "synchronizing": "Synchronizing all accounts...",
+    "synchronizing": "Synchronizing accounts with the blockchain: ",
```

Comment:
I forgot whether we are allowed to change values here. Only if they have not yet been translated? And maybe "with their [respective] blockchain"? Or shall we sacrifice accuracy for easier understanding?

Reply:
I agree that "with their respective blockchains" would be nice to be technically correct, but I think we should go for simplicity in this case. Also easier to translate.

Reply:
If we get a lot of hate for it from grammar nazis then we can change it :fearful:

Reply:
Changed back here and made the change in locize

Reply:
Great, thanks!